services/user/Branding/service/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+#[psibase::service(name = "branding")]
+mod service {
+    use async_graphql::{Object, SimpleObject};
+    use psibase::{
+        anyhow, check, get_sender, get_service, serve_content, serve_graphiql, serve_graphql,
+        serve_simple_ui, store_content, Fracpack, HexBytes, HttpReply, HttpRequest, SingletonKey,
+        Table, ToSchema, WebContentRow,
+    };
+    use serde::{Deserialize, Serialize};
+
+    #[table(name = "NetworkNameTable")]
+    #[derive(Fracpack, ToSchema, SimpleObject, Serialize, Deserialize, Debug)]
+    pub struct NetworkName {
+        pub name: String,
+    }
+
+    impl NetworkName {
+        #[primary_key]
+        fn by_key(&self) -> SingletonKey {
+            SingletonKey {}
+        }
+    }
+
+    impl Default for NetworkName {
+        fn default() -> Self {
+            NetworkName {
+                name: String::from("psibase"),
+            }
+        }
+    }
+
+    #[table(record = "WebContentRow", index = 1)]
+    struct WebContentTable;
+
+    #[action]
+    #[allow(non_snake_case)]
+    fn setNetworkName(name: String) {
+        NetworkNameTable::new()
+            .put(&NetworkName { name: name.clone() })
+            .unwrap();
+
+        Wrapper::emit().history().networkNameChanged(name);
+    }
+
+    #[event(history)]
+    pub fn networkNameChanged(name: String) {}
+
+    struct Query;
+
+    #[Object]
+    impl Query {
+        async fn network_name(&self) -> String {
+            let curr_val = NetworkNameTable::new().get_index_pk().get(&SingletonKey {});
+            match curr_val {
+                Some(val) => val.name,
+                None => String::from("psibase"),
+            }
+        }
+    }
+
+    #[action]
+    #[allow(non_snake_case)]
+    fn serveSys(request: HttpRequest) -> Option<HttpReply> {
+        None.or_else(|| serve_content(&request, &WebContentTable::new()))
+            .or_else(|| serve_graphql(&request, Query))
+            .or_else(|| serve_graphiql(&request))
+            .or_else(|| serve_simple_ui::<Wrapper>(&request))
```

Comment:
Are these needed when we have a full UI?

Reply:
graphiql, for debugging and whatnot (ensure that a failed call would pass if structured properly).
simple_ui? not really, but also could be useful for debugging.
Easy enough to add back in, I suppose.
It'd be useful to have an agreement among the team about when to include these or not.